target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8072-wax620.dts

```diff
@@ -63,47 +64,62 @@
 		compatible = "gpio-leds";
 
 		led_system_red: system-red {
-			label = "system:red";
+			function = LED_FUNCTION_FAULT;
+			color = <LED_COLOR_ID_RED>;
 			gpios = <&tlmm 54 GPIO_ACTIVE_HIGH>;
 		};
 
 		led_system_green: system-green {
-			label = "system:green";
+			function = LED_FUNCTION_STATUS;
+			color = <LED_COLOR_ID_GREEN>;
 			gpios = <&tlmm 55 GPIO_ACTIVE_HIGH>;
 		};
 
 		led_system_blue: system-blue {
-			label = "system:blue";
+			function = LED_FUNCTION_BOOT;
+			color = <LED_COLOR_ID_BLUE>;
 			gpios = <&tlmm 56 GPIO_ACTIVE_HIGH>;
 		};
 
-		led_lan_g {
+		led-lan-green {
 			label = "lan:green";
+			function = LED_FUNCTION_LAN;
+			color = <LED_COLOR_ID_GREEN>;
 			gpios = <&led_gpio 0 GPIO_ACTIVE_HIGH>;
 		};
 
-		led_lan_o {
+		led-lan-orange {
 			label = "lan:orange";
+			function = LED_FUNCTION_LAN;
+			color = <LED_COLOR_ID_ORANGE>;
 			gpios = <&led_gpio 1 GPIO_ACTIVE_HIGH>;
 		};
 
-		led_2g_b {
+		led-2g-blue {
 			label = "wlan2g:blue";
+			function = LED_FUNCTION_WLAN;
+			color = <LED_COLOR_ID_BLUE>;
 			gpios = <&led_gpio 2 GPIO_ACTIVE_HIGH>;
 		};
 
-		led_2g_g {
+		led-2g-green {
 			label = "wlan2g:green";
+			function = LED_FUNCTION_WLAN;
+			color = <LED_COLOR_ID_GREEN>;
 			gpios = <&led_gpio 3 GPIO_ACTIVE_HIGH>;
 		};
 
-		led_5g_b {
+		led-5g-blue {
 			label = "wlan5g:blue";
```

Comment:
Drop the label if you are converting to color/function

Reply:
The naming in Luci became useless without label, so reverting this.
But keeping the new led-name so it's similar between the wax-family.